luarules/gadgets/unit_paralyze_damage_limit.lua

```diff
@@ -130,3 +134,44 @@ function gadget:UnitPreDamaged(uID, uDefID, uTeam, damage, paralyzer, weaponID,
     end
     return damage, 1
 end
+
+--
+-- Special Stun Override: to restore spy bot vs anti-nuke building mechanic (20s stun vs new 8s stun)
```

Comment:
Terms like "new" or "restore" are quickly going to become meaningless and just bring needless context to anybody working with this in the future. If anything, explain the design intents behind the exception (it's desirable not because it was some "old, previous" mechanic but because it achieves goals X, Y and Z).
```suggestion
-- Special Stun Override: spy bot vs anti-nuke building mechanic (20s stun vs standard 8s stun)
```

Reply:
Updated text to be clearer and better explain what it does and what it could do.